bootstring/src/main/scala/cats/bootstring/Base.scala

```diff
@@ -101,7 +101,23 @@ object Base {
           s"Code point $codePoint is not valid for the given base.")
       }
 
-    unsafeFrom(36, d => lowercaseArray(d), d => uppercaseArray(d), codePointDigitToInt)
+    // Developer's note,
+    //
+    // This method exists because of some very strange behavior on ScalaJS
+    // where the (caught) index out of bounds exception caused an
+    // "UndefinedBehavior" exception to be thrown.
```

Comment:
This is part of Scala.js semantics:
https://www.scala-js.org/doc/semantics.html#undefined-behaviors

> Because Scala.js does not receive VM support to detect such erroneous conditions, checking them is typically too expensive.

Reply:
Ah, good to know! I'll update that comment then.